kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/ProcurementCardForm.java

```diff
@@ -29,6 +36,41 @@ public ProcurementCardForm() {
     public String getDocuwareTableParam() {
         return SpringContext.getBean(ParameterService.class).getParameterValueAsString(ProcurementCardDocument.class, KFSConstants.DOCUWARE_TABLE_PARAMETER);
     }
+    
+    @Override
+    public List<ExtraButton> getExtraButtons() {
+        extraButtons.clear();
+
```

Comment:
Optional refactoring: Maybe the logic to display the extra returnToReconcilerButton button in a service method returning boolean rather than in the form...